package.json

```diff
@@ -686,13 +686,21 @@
       "default": ""
     },
     {
-      "title": "OpenAI API URL",
-      "name": "openAIAPIURL",
+      "title": "OpenAI Endpoint",
+      "name": "openAIEndpoint",
```

Comment:
@tisfeng  I just noticed you have updated name property of OpenAI API URL option. I am afraid this will cause the endpoint already set by the user to be overwritten by the default value after the update, because it is actually a new  option and the old one was removed.

Reply:
Thank you reminding, I just fixed it 4907c2e